Tasks/range.js

```diff
@@ -0,0 +1,15 @@
+// Generate int array from given range
+
+;_range = ( ...Range ) => {;
```

Comment:
```suggestion
const rangeIntoArray = (...range) => {
```
improved naming, now we can understand what the function does, removed ';'